posthog/api/team.py

```diff
@@ -38,9 +41,20 @@ def has_permission(self, request: request.Request, view) -> bool:
         if request.method in CREATE_METHODS and (
             (user.organization is None)
             or (
-                user.organization.teams.exclude(is_demo=True).count() >= 1
+                # if we're not requesting to make a demo project
+                "is_demo" not in request.data
+                # if the org already has more than 1 non-demo project (need to be able to make the initial project)
+                and user.organization.teams.exclude(is_demo=True).count() >= 1
+                # and the org isn't allowed to make multiple projects
```

Comment:
Nitpicking: These interspersed comments are quite hard to read, it'd be a bit clearer to explain the group of `and`s in one go.